src/stores/store-urls-custom-admin.md

```diff
@@ -87,6 +90,21 @@ As a precaution, do not try to change the Admin URL yourself unless you know how
    {:.bs-callout-info}
    Note that the changes made using the Method 1 will have the priority over the changes made in the `app/etc/env.php` file.
 
+## Method 3: Change the Admin Path via CLI
```

Comment:
Since method 2 indicates the command line also, I suggest changing this header to be more specific.

```suggestion
## Method 3: Change the Admin Path using the Magento CLI
```